src/gui/editors/AutomationEditor.cpp

```diff
@@ -398,6 +398,58 @@ void AutomationEditor::drawLine( int x0In, float y0, int x1In, float y1 )
 
 
 
+bool AutomationEditor::fineTuneValue(timeMap::iterator node, bool editingInValue)
```

Comment:
If we change `editingInValue` to `editingOutValue` then we can simplify the switch statement in `mouseDoubleClickEvent`.